IBF-Typhoon-model/run_model_V2.R

```diff
@@ -168,7 +168,7 @@ df_impact_forecast <- as.data.frame(y_predicted) %>%
     region = substr(GEN_mun_code, 1, 4),
     Damaged_houses = as.integer(GEO_n_households * e_impact * 0.01),
   ) %>%
-  filter(WEA_dist_track < 500) %>%
+  filter(WEA_dist_track < 100) %>%
```

Comment:
please add this line after line 38 (to be efficient with one commit) -- not sure how this is working fine in linux but the geometry of the file has some invalid geometries (the following line will fix it )
php_admin3<-st_make_valid(php_admin3)
php_admin1<-st_make_valid(php_admin1)